tests/Laravel/ServiceProviderTest.php

```diff
@@ -162,6 +181,19 @@ public function testListenerWithLocaleUpdatedClass()
         $service->setAppGetter(static fn () => null);
         $service->updateLocale();
         $this->assertSame('ckb', Carbon::getLocale());
+
+        $service->setAppGetter(static fn () => $app);
+        $this->assertSame('en', Carbon::getFallbackLocale());
```

Comment:
I actually test here that event NOT having effect on fallback locale. Dunno if it makes any sense😅 /3